src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -634,7 +636,7 @@ private void AddServicesProducedInfo(IResource modelResource, IAnnotationHolder
             }
             catch (Exception ex)
             {
-                Console.WriteLine($"Could not stop {resourceName} '{res.Metadata.Name}': {ex}");
+                _logger.LogInformation("Could not stop {resourceName} '{res.Metadata.Name}': {ex}", resourceName, res.Metadata.Name, ex);
```

Comment:
Well, the call stack shows `TaskCanceledException`; I assumed if that was a call failure because DCP was not available, the exception would be different/network-related.